cryptographic_estimators/MQEstimator/MQAlgorithms/bjorklund.py

```diff
@@ -186,16 +210,31 @@ def _find_optimal_tilde_o_parameters(self):
         """
         self._optimal_parameters['lambda_'] = 0.19677
 
-    @staticmethod
+    @staticmethod    
     def _internal_time_complexity_(n: int, m: int, lambda_: float):
         """
         Helper function. Computes the runtime of the algorithm for given n, m and lambda
         """
-        if n <= 1:
+        # uses brute force
+        if n <= 1:            
             return 1
-        else:
-            l = floor(lambda_ * n)
-            T1 = (n + (l + 2) * m * sum_of_binomial_coefficients(n, 2) +
-                  (n - l) * 2 ** (n - l))
-            s = 48 * n + 1
-            return s * sum_of_binomial_coefficients(n - l, l + 4) * (Bjorklund._internal_time_complexity_(l, l + 2, lambda_) + T1)
+        T = 0  # Time complexity
+        l = floor(lambda_ * n)  # n1, l from Björklund.
+        s = 48 * n + 1  # number of iterations
+        sumbin_n_2 = sum_of_binomial_coefficients(n, 2)
+        sumbin_B = sum_of_binomial_coefficients(n - l, l + 4)
+        T += 2 ** (n-l)  # Line 5, initialize array of size 2^{n-l}
```

Comment:
What Line 5 refers to? A pseudocode in a paper? Please add a reference to is in the dumentation corresponding to the method`_internal_time_complexity_`.